lib/spack/spack/build_environment.py

```diff
@@ -109,7 +109,7 @@ def __call__(self, *args, **kwargs):
         disable = env_flag(SPACK_NO_PARALLEL_MAKE)
         parallel = not disable and kwargs.get('parallel', self.jobs > 1)
 
-        if parallel:
+        if parallel and 'bazel' not in self.command:
```

Comment:
Hmm, the only difference between `MakeExecutable` and `Executable` is that `MakeExecutable` can specify a certain number of jobs. Why would you create a new subclass?